webdriver/tests/support/fixtures.py

```diff
@@ -195,13 +195,15 @@ async def bidi_session(capabilities, configuration, request):
     await reset_current_session_if_necessary(caps, True)
 
     if _current_session is None:
-        _current_session = webdriver.BidiSession(
+        _current_session = webdriver.Session(
             configuration["host"],
             configuration["port"],
-            capabilities=caps)
-    try:
-        await _current_session.start()
+            capabilities=caps,
+            enable_bidi=True)
 
+    try:
+        _current_session.start()
+        await _current_session.bidi_session.start()
```

Comment:
Can't `_current_session.start()` handle that implicitly?

Reply:
It seems more flexible to allow creating a session and then later connecting to the bidi session (e.g. if you wanted to test that element ids created before the bidi connection were valid over the bidi connection). So I think this is a better approach.